dev/helm-local.yml

```diff
@@ -39,7 +39,7 @@ engine:
   replicaCount: 1
 celery:
   replicaCount: 1
-  worker_beat_enabled: false
+  worker_beat_enabled: true
```

Comment:
I think this should be enabled by default for local development (took me several minutes to figure out why `apps.heartbeat.tasks.check_heartbeats` wasn't running)